packages/cli/src/utils/validate-fs.ts

```diff
@@ -50,22 +48,21 @@ export async function validateFilePath(filePath: string): Promise<boolean> {
 }
 
 /**
- * Validates a directory path.
- * Throws on validation failure.
+ * Validates a directory path. Throws on validation failure.
  *
- * @param dirPath - The directory path to validate
- * @param dirName
- * @param createDir - Whether to create the directory if it doesn't exist
- * @returns True if validation succeeded
+ * @param dirPath - The directory path to validate.
+ * @param createDir - Whether to create the directory if it doesn't exist.
+ * @returns `true` if validation succeeded or the directory was created, and
```

Comment:
This never returns `false` as it throws instead.

Reply:
3f8379b